codebuild/spec/buildspec_ubuntu_integrationv2.yml

```diff
@@ -29,7 +29,7 @@ batch:
         variables:
           S2N_BUILD_PRESET:
             - awslc_gcc9
-            - awslc-fips_gcc4-8
+            - awslc-fips-2022_gcc4-8
```

Comment:
Is this a correct swap? If I'm reading right, without a version fips was latest-ish fips: "last done on 2023-02-22." Should this be "latest" or at least 2024, to reflect the intent?

Reply:
Without a version the current script uses the FIPS 2021 v1.x branch. My intent is to deprecate that version and 
I'm making the leap that anyone with a compiler that old isn't going to be on latest, so that leaves 2022.